diff --git a/src/lib/connect.c b/src/lib/connect.c
index 854e994..2af5449 100644
--- a/src/lib/connect.c
+++ b/src/lib/connect.c
@@ -178,7 +178,7 @@ LIBC_CONNECT_RET_TYPE tsocks_connect(LIBC_CONNECT_SIG)
                 * .onion cookie address that is by default in the loopback network
                 * thus this check is done after the onion entry lookup.
                 */
-               if (utils_sockaddr_is_localhost(addr)) {
+               if (0 && utils_sockaddr_is_localhost(addr)) {
                        /*
                         * Certain setups need to be able to reach localhost, despite
                         * running torsocks. If they enabled the config option, allow such
diff --git a/src/lib/torsocks.c b/src/lib/torsocks.c
index 16f2da0..324fc10 100644
--- a/src/lib/torsocks.c
+++ b/src/lib/torsocks.c
@@ -549,7 +549,7 @@ int tsocks_tor_resolve(int af, const char *hostname, void *ip_addr)
         * reserved IP address that acts as a cookie that we will use to find the
         * onion hostname at the connect() stage.
         */
-       if (utils_strcasecmpend(hostname, ".onion") == 0) {
+       if (0 && utils_strcasecmpend(hostname, ".onion") == 0) {
                struct onion_entry *entry;
 
                entry = get_onion_entry(hostname, &tsocks_onion_pool);

